htdocs/js/util/queryString.js

```diff
@@ -50,15 +50,6 @@ var QueryString = {
     // Deep copy of object representation of query
     var queryStringObj = JSON.parse(JSON.stringify(currentQuery));
 
-    // Make sure that both key and value are of string type
-    if (typeof fieldName !== "string" || typeof fieldValue !== "string") {
```

Comment:
since the only expected change in type is for fieldValue, the check on fieldName is still valid. 
i would either:
a) remove `|| typeof fieldValue !== "string"` 
or
b) change it to `|| (typeof fieldValue !== "string" && typeof fieldValue !== "object")` 
and then update the error message accordingly. 

the documentation in the comment above should also be amended to suggest that fieldValue can be either string or object. 
same comment applies to the changes in FilterForm.js